nidirect_gp/templates/node--gp-practice--teaser.html.twig

```diff
@@ -0,0 +1,35 @@
+{#
+/**
+ * @file
+ * Default template for GP Practice's teaser view mode.
+ * USAGE: copy to your theme folder templates directory, or
+ * use from a module with a suitable hook_theme instance.
+ */
+#}
+{%
+  set classes = [
+  view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+  'clearfix',
+]
+%}
+<h3><a href="{{ url }}">{{ label }}</a></h3>
+<div class="gp-practice-members">
+  {% if content.lead_gp %}
```

Comment:
Need to render the content array to ensure the cache tags bubble up.

Reply:
Thanks. Now sorted with final call to render 'content' passed via the `without` filter where some fields have been selectively used.